### ISSUE TITLE
Will a larger value for partials_n_frames be better?

### ISSUE THREAD

**Issue by Coastchb** [CorentinJ/Real-Time-Voice-Cloning#403]
Hi, all.

In theory, using more frames as input to the model will include more information about the speaker.
So,  I set the partials_n_frames to 160(exp1) and 300(exp2) respectively, and trained on the same training set.
On training set, the loss and EER of exp2 is slightly lower than that of exp1.But on test set (we record with phones), EER is higher on exp2.

So, I am really confused:
1. partials_n_frames=160 is the best configuration? Did you try any other values?
2. EER is lower on training, but higher on test set. It seems that it is not caused by overfitting. In fact, on other experiments (eg. Bi-LSTM instead of Uni-LSTM), this also occurs.

Do you have any advice?

**Comment by blue-fish** [comment]
@Coastchb If you still need an answer please try asking your question in #364 . I am closing this issue due to inactivity, you are welcome to reopen it at any time.
